config/rootfs/chromiumos/Changelog.md

```diff
@@ -0,0 +1,39 @@
+<img src="https://kernelci.org/image/kernelci-horizontal-color.png"
+     alt="KernelCI project logo"
+     width="40%" />
+
+# KernelCI ChromiumOS image changelog
+
+This file tracks divergences between the KernelCI built ChromiumOS images and upstream ChromiumOS as well as any changes between image relases.
+
+Divergences can be considered tech-debt and in the long run need to be kept under control and minimized, therefore this chanelog should reflect its evolution from version to version.
+
+## Release 106
+
+### Changes since previous version (R100)
+	- A custom repo manifest is used to build images which points to forked repositories.
+	- SElinux is now disabled in userspace, see issue #1372.
```

Comment:
I think better to specify full URL to issue https://github.com/kernelci/kernelci-core/issues/1372